docs/overview/architecture.md

```diff
@@ -113,4 +113,32 @@ When the user searches a vector from Vald:
 
 <!-- ### Update -->
 
+When the user updates a vector from Vald:
+
+1. Vald Ingress receives the request from the user. The request includes the vector and the vector ID.
```

Comment:
Is it includes the old vector ID, and the new vector?

Reply:
yes. it is.

Reply:
but, I can not understand about old vector ID.
VectorID is not re-generated. so I think vector ID is correct.

Reply:
Is it correct?
```suggestion
1. Vald Ingress receives the request from the user. The request includes the new vector and the vector ID to be update.
```

Reply:
thank you :bow: